respa_admin/templates/respa_admin/resources/form/_period_template.html

```diff
@@ -1,28 +1,32 @@
 {% load i18n %}
-
+{% for form in period_template_formset %}
```

Comment:
Period templates are added here. Adding the forms inside a <template> ensures they are not included with the other form fields, and can be copied to the period list using the existing JS functionality.